src/Components/Main/TodoList/TodoList.tsx

```diff
@@ -0,0 +1,45 @@
+import React, { useContext } from 'react';
+import { TodosContext } from '../../../Context/TodosContext';
+import { Status } from '../../../Types/Status';
+import Todos from '../../../Types/Todos';
+import { TodoItem } from '../TodoItem/TodoItem';
+
+export const TodoList: React.FC = () => {
+  const {
+    todos,
+    status,
+  } = useContext(TodosContext);
+
+  let filterTodos = todos;
+
+  const activeFilter = filterTodos.filter((todo: Todos) => !todo.completed);
+  const completedFilter = filterTodos.filter((todo: Todos) => todo.completed);
+
+  switch (status) {
+    case Status.all:
```

Comment:
enum's items should be in camelCase or UPPER_CASE
```suggestion
    case Status.All:
```